docs/import/upload_image_collection.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Upload Image Collection</title>
+  <!-- Load jQuery, a 3rd-party library. -->
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+  <!-- TODO(janakr): Switch back to compiled version in production. -->
+  <!-- TODO(janakr): This is pinned to a particular version, but we should be targeting the live
+       version, whatever it is. -->
+  <script src="https://rawcdn.githack.com/google/earthengine-api/3bb86bfc4f3d9eed98220f3d225b414982915b86/javascript/build/ee_api_js_debug.js"></script>
+  <script src="https://apis.google.com/js/api.js"></script>
+
+  <script src="../external_libs/firebase-app-7.6.1.js"></script>
+  <script src="../external_libs/firebase-firestore-7.6.1.js"></script>
+  <script src="../external_libs/firebase-auth-7.6.1.js"></script>
+
+  <script type="module" src="./upload_image_collection_startup.js"></script>
+
+  <!-- Load our styles. -->
+  <link rel="stylesheet" href="../style.css">
+</head>
+<body>
+
+<div id="navbar"></div>
```

Comment:
I don't see a navbar showing up on the page - not supposed to be hooked up yet?

Also don't see a header though there appears to be space for one.

Reply:
Because of missing startup file. Should be fixed.